web-console/odc-about-developer-perspective-in-the-web-console.adoc

```diff
@@ -0,0 +1,32 @@
+ifdef::context[:parent-context-of-odc-about-developer-perspective-in-the-web-console: {context}]
+[id="odc-about-developer-perspective-in-the-web-console_{context}"]
+= About Developer perspective in the web console
+include::modules/common-attributes.adoc[]
+:context: odc-about-developer-perspective
+toc::[]
+
+The {product-title} web console provides two perspectives; the *Administrator* perspective and the *Developer* perspective.
+
+The *Developer* perspective provides the following developer focused features:
```

Comment:
developer-focused

Reply:
The *Developer* perspective provides the ability to:
    
    * Create and deploy ...
    * Visually interact ... (and so on)